src/Module/Hostxrd.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Renderer;
+use Friendica\Protocol\Salmon;
+use Friendica\Util\Crypto;
+
+/**
+ * Prints the host-meta text
+ */
+class Hostxrd extends BaseModule
```

Comment:
Please use a human readable name for this module and use the router to match the expected query path.

Reply:
OK, so it's ready to rumble, perfect! :D

Reply:
Yep, the admin PR is building on top of it, but it's definitely ready.

Reply:
I think that using any other name would only be confusing here (with possibly the exception of calling it "HostXRD", since "XRD" looks like some acronym.

I don't think that we should try to force renaming this class just because we can now. It is also for practical reasons. I don't like to have to look into the router stuff every time just because I can't easily guess the correct name.

Reply:
HostXRD would be an improvement, but this form isn't available with the module auto-discovery feature that requires a single uppercase letter at the start and only lowercase letter after that.